*** Reading local file: /opt/airflow/logs/func/send_tab/2022-02-25T20:59:49.556540+00:00/1.log
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: func.send_tab manual__2022-02-25T20:59:49.556540+00:00 [queued]>
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: func.send_tab manual__2022-02-25T20:59:49.556540+00:00 [queued]>
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-25, 21:00:06 UTC] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): send_tab> on 2022-02-25 20:59:49.556540+00:00
[2022-02-25, 21:00:06 UTC] {standard_task_runner.py:52} INFO - Started process 7182 to run task
[2022-02-25, 21:00:06 UTC] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'func', 'send_tab', 'manual__2022-02-25T20:59:49.556540+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/test_task_dag.py', '--cfg-path', '/tmp/tmppkvr4g0o', '--error-file', '/tmp/tmp98jht9zn']
[2022-02-25, 21:00:06 UTC] {standard_task_runner.py:77} INFO - Job 103: Subtask send_tab
[2022-02-25, 21:00:07 UTC] {logging_mixin.py:109} INFO - Running <TaskInstance: func.send_tab manual__2022-02-25T20:59:49.556540+00:00 [running]> on host a8a367e07e36
[2022-02-25, 21:00:07 UTC] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=func
AIRFLOW_CTX_TASK_ID=send_tab
AIRFLOW_CTX_EXECUTION_DATE=2022-02-25T20:59:49.556540+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-25T20:59:49.556540+00:00
[2022-02-25, 21:00:08 UTC] {python.py:175} INFO - Done. Returned value was: None
[2022-02-25, 21:00:08 UTC] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=func, task_id=send_tab, execution_date=20220225T205949, start_date=20220225T210006, end_date=20220225T210008
[2022-02-25, 21:00:08 UTC] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-25, 21:00:08 UTC] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check

